### PR TITLE
Temporary changes for outreach events collection and hiding the register button from future events

### PR DESCRIPTION
**Pull request by SwathiSelvakumaran**
…ter button in future events

### CODE REVIEW

File: app/src/main/res/layout/bottom_sheet_event.xml [hunk 1]
```diff
@@ -296,7 +297,9 @@
                     android:text="@string/sign_up"
                     android:textAllCaps="false"
                     android:textColor="@color/accent_yellow"
-                    android:textStyle="bold">
+                    android:textStyle="bold"
+                    android:visibility="gone"
+                    tool:visibility="gone">
```
Comment (phillipsk) [inline app/src/main/res/layout/bottom_sheet_event.xml:302]: tools:visibility="visible" so developers know it's there

File: app/src/main/res/layout/bottom_sheet_event.xml [hunk 2]
```diff
@@ -173,7 +173,8 @@
                     android:textColor="@color/accent_yellow"
                     android:textStyle="bold"
                     android:layout_marginStart="5dp"
-                    android:visibility="gone">
+                    android:visibility="gone"
+                    tool:visibility="gone">
```
Comment (phillipsk) [inline app/src/main/res/layout/bottom_sheet_event.xml:177]: tools:visibility="visible" so developers know it's there

**Review by phillipsk — APPROVED** [review]

File: app/src/main/res/layout/bottom_sheet_event.xml (hunk 1 above)
Comment (SwathiSelvakumaran) [reply app/src/main/res/layout/bottom_sheet_event.xml]: Updated

File: app/src/main/res/layout/bottom_sheet_event.xml (hunk 2 above)
Comment (SwathiSelvakumaran) [reply app/src/main/res/layout/bottom_sheet_event.xml]: Updated

**Pull request merged**